### DISCUSSION TITLE
ResizeObserver on old iOS

### DISCUSSION THREAD

**Discussion by jnm** [outline/outline#3627] [General] [answered]
Hello,

I have someone on my team using a pre-iOS-13.4 iPad with [no support](https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver#browser_compatibility) for `ResizeObserver`. When they try to edit a document, they get the "Something Unexpected Happened" message, with the detail being `ReferenceError: Can't find variable: ResizeObserver`.

Would the developers entertain adding support for these older devices, or accepting a PR that provides a workaround for missing `ResizeObserver`?

Thanks for reading!

[edit: for reference, this was fixed in #3629. thanks, Tom!]

**Comment by tommoor** [comment, marked as answer]
Definitely open to a polyfill! But it'll need to be async-loaded to not increase the size of the bundle for all other users… [ref](https://github.com/rehooks/component-size#polyfill)

**Reply by tommoor** [reply]
I'll do it now actually

**Reply by jnm** [reply]
:heart_eyes:

**Reply by tommoor** [reply]
If your team is using the cloud version this fix has been deployed.
